composer.json

```diff
@@ -18,20 +18,20 @@
         "ext-mbstring": "*",
         "behat/gherkin": "^4.5.1",
         "behat/transliterator": "^1.2",
-        "symfony/console": "~2.5||~3.0",
-        "symfony/config": "~2.3||~3.0",
-        "symfony/dependency-injection": "~2.1||~3.0",
-        "symfony/event-dispatcher": "~2.1||~3.0",
-        "symfony/translation": "~2.3||~3.0",
-        "symfony/yaml": "~2.1||~3.0",
-        "symfony/class-loader": "~2.1||~3.0",
+        "symfony/console": "~2.5||~3.0||~4.0",
+        "symfony/config": "~2.3||~3.0||~4.0",
+        "symfony/dependency-injection": "~2.1||~3.0||~4.0",
+        "symfony/event-dispatcher": "~2.1||~3.0||~4.0",
+        "symfony/translation": "~2.3||~3.0||~4.0",
+        "symfony/yaml": "~2.1||~3.0||~4.0",
+        "symfony/class-loader": "~2.1||~3.0||~4.0",
         "psr/container": "^1.0",
         "container-interop/container-interop": "^1.2"
     },
 
     "require-dev": {
-        "symfony/process": "~2.5|~3.0",
-        "phpunit/phpunit": "^4.8.36|^6.3",
+        "symfony/process": "~2.5|~3.0|~4.0",
+        "phpunit/phpunit": "^4.8.36|^6.3"
```

Comment:
you're missing a comma here

Reply:
I was so pleased about resolving the conflict via github, too